src/wp-includes/theme-compat/header.php

```diff
@@ -18,44 +18,31 @@
 <!DOCTYPE html>
 <html <?php language_attributes(); ?>>
 <head>
-<link rel="profile" href="https://gmpg.org/xfn/11" />
-<meta http-equiv="Content-Type" content="<?php bloginfo( 'html_type' ); ?>; charset=<?php bloginfo( 'charset' ); ?>" />
-
-<title><?php echo wp_get_document_title(); ?></title>
-
-<link rel="stylesheet" href="<?php bloginfo( 'stylesheet_url' ); ?>" type="text/css" media="screen" />
-<link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>" />
-
-<?php if ( file_exists( get_stylesheet_directory() . '/images/kubrickbgwide.jpg' ) ) { ?>
-<style type="text/css" media="screen">
-
+	<meta charset="<?php bloginfo( 'charset' ); ?>" />
+	<title><?php echo wp_get_document_title(); ?></title>
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<link rel="profile" href="https://gmpg.org/xfn/11" />
+	<link rel="stylesheet" href="<?php echo esc_url( get_stylesheet_uri() ); ?>" type="text/css" media="all" />
```

Comment:
stylesheet link uses `get_stylesheet_uri` function, as Twenty Ten and Twenty Eleven do, and this changes `media` to all